### ISSUE TITLE
CAPTCHA: Error message fails contrast ratio

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#168]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** The error message (see screenshot below) fails contrast ratio over its background.

Foreground= #ff0000
Background = #f9f9f9

![image](https://user-images.githubusercontent.com/12256067/162527501-8956492c-bec4-4a04-8535-145eb417bdb4.png)

**Steps to reproduce:**

1. Go to the above URL
2. Expand the Report a problem or mistake on this page component
3. Select the checkbox under CAPTCHA
4. Wait for the error message to appear

**Expected results:** The contrast ratio for regular text should be at least 4.5:1 over its background.

**Comment by echouin** [comment]
Believe that this is fixed.

**Comment by YuxuanXie01** [comment]
Retested, issue persists. The text colour (#FF0000) on the background (#F9F9F9) has a contrast ratio of 3.8:1, which does not meet the standard 4.5:1. 
![image](https://github.com/user-attachments/assets/5bfd3a6b-feec-404f-a81d-86cf3575aa61)

**Comment by 40999221** [comment]
Issue is still exist, color contrast ratio for error message text is 3.8:1 against 4.5:1 of WCAG standards.

![image](https://github.com/user-attachments/assets/4cceb852-b69c-4787-8c76-9459e34c7826)

**Comment by zimw7** [comment]
Issue still exists. The text color (#FF0000) on the background (#F9F9F9) has a contrast ratio of 3.8:1. It should be at least 4.5:1 for regular text.
![image](https://github.com/user-attachments/assets/b3814091-b75d-450c-b59e-2dff1e407e4c)

**Comment by mmiron-HC** [comment]
@40999221 Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605) Put the status of this one to Escalated and please include a note to say that this is a google product limitation. Add product_limitation to the labels as well
